internal/tools/tools.go

```diff
@@ -38,6 +39,16 @@ type Manifest struct {
 	Parameters  []ParameterManifest `json:"parameters"`
 }
 
+// Definition for a tool the MCP client can call.
+type McpManifest struct {
+	// The name of the tool.
+	Name string `json:"name"`
+	// A human-readable description of the tool.
+	Description string `json:"desciprtion,omitempty"`
```

Comment:
nit: just following along with MCP changes and noticed a typo 😄 
```suggestion
	Description string `json:"description,omitempty"`
```

Reply:
Thanks for catching this! haha